static/css/main.css

```diff
@@ -4,138 +4,109 @@
 
 
 html, body {
-  height: 100%;
-  margin: 0;
-  padding: 0;
+    height: 100%;
+    margin: 0;
+    padding: 0;
 }
 body {
-  font-family: 'Open Sans', Helvetica, Arial, sans-serif;
-  background: #000;
-  color: #bdbdbd;
+    font-family: 'Open Sans', Helvetica, Arial, sans-serif;
+    fill: #fff;
+    background: #000;
+    color: #bdbdbd;
 }
 
 .container {
-  bottom: 0;
-  display: block;
-  height: 100%;
-  left: 0;
-  margin: 0;
-  padding: 0;
-  top: 45px;
-  width: 100%;
-  z-index: 2;
-}
-
-#container.announcing .customTitle {
-  color: orange;
-}
-
-#container.alarming.urgent .customTitle {
-  color: red;
-}
-
-#container.alarming.warning .customTitle {
-  color: yellow;
+    bottom: 0;
+    display: block;
+    height: 100%;
+    left: 0;
+    margin: 0;
+    padding: 0;
+    top: 45px;
+    width: 100%;
+    z-index: 2;
 }
 
-.primary {
-  font-family: 'Ubuntu', Helvetica, Arial, sans-serif;
-  vertical-align: middle;
-  clear: both;
+.status {
+    font-family: 'Ubuntu', Helvetica, Arial, sans-serif;
+    height: 180px;
+    vertical-align: middle;
+    clear: both;
 }
 
 .bgStatus {
-  float: right;
-  text-align: center;
-  white-space: nowrap;
-  padding-right: 20px;
+    float: right;
+    text-align: center;
+    white-space: nowrap;
+    padding-right: 20px;
 }
 
-.urgent .bgButton {
+.bgButton.urgent {
   color: red;
 }
 
-.warning .bgButton {
+.bgButton.warning {
   color: yellow;
 }
 
-.inrange .bgButton {
+.bgButton.inrange {
   color: #4cff00;
 }
 
 .bgStatus .currentBG {
-  font-size: 120px;
-  line-height: 100px;
-  text-decoration: line-through;
-  vertical-align: middle;
+    font-size: 120px;
+    line-height: 100px;
+    text-decoration: line-through;
+    vertical-align: middle;
 }
 
 .bgStatus .currentBG.bg-limit, .bgStatus .currentBG.icon-hourglass {
   font-size: 90px;
 }
 
-.bgStatus .pill.direction {
-  font-weight: normal;
-  text-decoration: none;
-  font-size: 90px;
-  line-height: 90px;
-  vertical-align: middle;
-  border: none;
-  background: none;
-}
-
-.bgStatus .majorPills {
-  font-size: 30px;
-  line-height: 34px;
-}
-
-.minorPills {
-  font-size: 22px;
-  line-height: 25px;
-  z-index: 500;
-  white-space: normal;
+.bgStatus .currentDirection {
+    font-weight: normal;
+    text-decoration: none;
+    font-size: 90px;
+    line-height: 90px;
+    vertical-align: middle;
 }
 
-.majorPills > span:not(:first-child) {
-  margin-left: 5px;
+.bgStatus .currentDetails {
+    font-size: 30px;
 }
 
-.minorPills > span:not(:first-child) {
-  margin-left: 5px;
+.currentDetails > span:not(:first-child) {
+    margin-left: 5px;
 }
 
 .pill {
-  white-space: nowrap;
-  border-radius: 5px;
-  border: 2px solid #808080;
-  background: #808080;
-  display: inline-block;
-  margin-bottom: 4px;
-  padding: 2px 0;
+    white-space: nowrap;
+    border-radius: 5px;
+    border: 2px solid #808080;
 }
 
-.pill em, .pill label {
-  padding: 2px;
-  white-space: nowrap;
+.pill * {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The universal selector (*) is known to be slow.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/969293254/issues/source?bid=2452374&fileBranchId=2861014#l89)